### PR TITLE
[Doc] Update document

### PR DESCRIPTION
**Pull request by gichan-jang**
I added feature 'Planned' for better understanding.

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

### CODE REVIEW

**Comment by wooksong** [comment]
You need to fill the body of your commit. By the way, what does `Dox` stand for? Documents?

**Comment by gichan-jang** [comment]
> You need to fill the body of your commit. By the way, what does `Dox` stand for? Documents?

Thnak you for your comments. 
I will add body of commit and 'Dox' is a typing error of 'Doc'..

File: Documentation/component-description.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Note that except for tensor\_decoder, which accepts data semantics from pipeline
 
 # Gstreamer Elements (Plugins)
 
-Note that "stable" does not mean that it is complete. It means that it has enough test cases and complies with the overall design; thus, "stable" features probably won't be modified extensively. Features marked "experimental" can be modified extensively due to its incomplete design and implementation or crudeness.
+Note that "stable" does not mean that it is complete. It means that it has enough test cases and complies with the overall design; thus, "stable" features probably won't be modified extensively. "experimental" can be modified extensively due to its incomplete design and implementation or crudeness. "planned" is still in the works so it will be released soon.
```
Comment (myungjoo) [inline Documentation/component-description.md]: ```s/"experimental"/"Experimental"/```
```s/"planned"/"Planned"/```

File: Documentation/component-description.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Note that except for tensor\_decoder, which accepts data semantics from pipeline
 
 # Gstreamer Elements (Plugins)
 
-Note that "stable" does not mean that it is complete. It means that it has enough test cases and complies with the overall design; thus, "stable" features probably won't be modified extensively. Features marked "experimental" can be modified extensively due to its incomplete design and implementation or crudeness.
+Note that "stable" does not mean that it is complete. It means that it has enough test cases and complies with the overall design; thus, "stable" features probably won't be modified extensively. Features marked "experimental" can be modified extensively due to its incomplete design and implementation or crudeness. "Planned" is still in the works so it will be released soon.
```
Comment (wooksong) [inline Documentation/component-description.md]: s/experimental/Experimental

**Pull request merged**